lib/pwwka/receiver.rb

```diff
@@ -12,7 +12,7 @@ class Receiver
     def initialize(queue_name, routing_key, prefetch: Pwwka.configuration.default_prefetch)
       @queue_name        = queue_name
       @routing_key       = routing_key
-      @channel_connector = ChannelConnector.new(prefetch: prefetch)
+      @channel_connector = ChannelConnector.new(prefetch: prefetch, connection_name: "s: #{queue_name}")
```

Comment:
The prefix `s` indicates that this connection is from a subscriber (as apposed to a publisher)